lib/ui/screens/accountScreen.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:flutter/material.dart';
+
+class AccountScreen extends StatefulWidget {
+  @override
+  _AccountScreenState createState() => _AccountScreenState();
+}
+
+class _AccountScreenState extends State<AccountScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      child: Container(
+        color: Colors.white,
+        child: Column(
+          children: <Widget>[
+            Container(
+              padding:
+                  const EdgeInsets.symmetric(vertical: 64.0, horizontal: 32.0),
+              color: Colors.greenAccent,
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: <Widget>[
+                  Row(
+                    children: <Widget>[
+                      Container(
+                          width: 100.0,
+                          height: 100.0,
+                          decoration: new BoxDecoration(
+                              shape: BoxShape.circle,
+                              image: new DecorationImage(
+                                  fit: BoxFit.fill,
+                                  image: new AssetImage(
+                                      "assets/account_profile.jpg")))),
```

Comment:
Also I probably forgot to explain this , but we store `"assets/account_profile.jpg"` as a const string in the strings file
then just call the variable name . So that it's reusable @mikeyyg96